packages/multicast_dns/lib/multicast_dns.dart

```diff
@@ -147,15 +147,8 @@ class MDnsClient {
       throw StateError('Cannot stop mDNS client while it is starting.');
     }
 
-    for (final RawDatagramSocket socket in _sockets) {
-      socket.close();
-    }
-    _sockets.clear();
-
-    for (final RawDatagramSocket socket in _toBeClosed) {
-      socket.close();
-    }
-    _toBeClosed.clear();
+    _incomingIPv4?.close();
+    _incomingIPv6?.close();
 
```

Comment:
Set to null after closing, in case someone calls `start` again.

Reply:
`_incomingIPv4` and `_incomingIPv6` ?